### DISCUSSION TITLE
Total Rain

### DISCUSSION THREAD

**Discussion by jwgizmo** [briis/hass-weatherflow2mqtt#169] [Q&A]
Is there a way to make a card to display the total rain over time? Like a week, month, quarter, or year - in numerical value(s). 
Something like this? (My data in empty)

![image](https://user-images.githubusercontent.com/28872913/179450830-c19f1744-0ecb-4a43-a5a6-accd4df2bdd8.png)

**Comment by GlennGoddard** [comment]
You can set up a history graph and change the number of hours for what you would like to show.
"Rain Today", you may have to adjust the length of data you store in your HA setup though to save and display more data.
I only save the last 14 days so that is all I can display in HA without importing it back with something like Grafana & InfluxDB.

So, yes it is possible, but may require some more work.

**Comment by jwgizmo** [comment]
Thank you, I’ll dig into it a bit deeper.
